32-bit Integration Tests failing to upload servicehub logs

```
Extra run diagnostics for logging, did not impact run results
Test execution time: 00:44:07.7645168
All tests passed
Copying ServiceHub logs to F:\workspace\_work\1\s\artifacts\log\Debug
##[error](NETCORE_ENGINEERING_TELEMETRY=Test) Tests failed
Cannot find path 'F:\workspace\_work\1\s\artifacts\tmp\Debug\servicehub\logs' because it does not exist.
System.Management.Automation.ItemNotFoundException: Cannot find path 'F:\workspace\_work\1\s\artifacts\tmp\Debug\servicehub\logs' because it does not exist.
   at System.Management.Automation.LocationGlobber.ExpandMshGlobPath(String path, Boolean allowNonexistingPaths, PSDriveInfo drive, ContainerCmdletProvider provider, CmdletProviderContext context)
   at System.Management.Automation.LocationGlobber.ResolveDriveQualifiedPath(String path, CmdletProviderContext context, Boolean allowNonexistingPaths, CmdletProvider& providerInstance)
   at System.Management.Automation.LocationGlobber.GetGlobbedMonadPathsFromMonadPath(String path, Boolean allowNonexistingPaths, CmdletProviderContext context, CmdletProvider& providerInstance)
   at System.Management.Automation.LocationGlobber.GetGlobbedProviderPathsFromMonadPath(String path, Boolean allowNonexistingPaths, CmdletProviderContext context, ProviderInfo& provider, CmdletProvider& providerInstance)
   at System.Management.Automation.SessionStateInternal.CopyItem(String[] paths, String copyPath, Boolean recurse, CopyContainers copyContainers, CmdletProviderContext context)
   at Microsoft.PowerShell.Commands.CopyItemCommand.ProcessRecord()
at TestUsingRunTests, F:\workspace\_work\1\s\eng\build.ps1: line 409
at <ScriptBlock>, F:\workspace\_work\1\s\eng\build.ps1: line 628
at <ScriptBlock>, F:\workspace\_work\_temp\c3e5413e-c186-4c35-bdfc-fb5b7af30a3e.ps1: line 3
at <ScriptBlock>, <No file>: line 1
Killing running build processes...
Killing running build processes...
##[error]PowerShell exited with code '1'.
```

```
##[error]Publishing build artifacts failed with an error: Not found PathtoPublish: F:\workspace\_work\1\s\artifacts\bin\Microsoft.VisualStudio.LanguageServices.IntegrationTests\Debug\net472\xUnitResults
```

See <https://dev.azure.com/dnceng/public/_build/results?buildId=952611&view=results>

--- a/eng/build.ps1
+++ b/eng/build.ps1
@@ -420,8 +420,13 @@ function TestUsingRunTests() {
     }
 
     if ($testVsi) {
-      Write-Host "Copying ServiceHub logs to $LogDir"
-      Copy-Item -Path (Join-Path $TempDir "servicehub\logs") -Destination (Join-Path $LogDir "servicehub") -Recurse
+      $serviceHubLogs = Join-Path $TempDir "servicehub\logs"
+      if (Test-Path $serviceHubLogs) {
+        Write-Host "Copying ServiceHub logs to $LogDir"
+        Copy-Item -Path $serviceHubLogs -Destination (Join-Path $LogDir "servicehub") -Recurse
+      } else {
+        Write-Host "No ServiceHub logs found to copy"
+      }
     }
   }
 }

